modules/node_modules/@ncigdc/components/SSM.js

```diff
@@ -210,23 +205,13 @@ const Mutation = compose(
             alignSelf: 'flex-start',
           }}
         />
-        {externalDbIds && !!Object.keys(externalDbIds).length &&
+        {dbSNP && /rs([0-9]+)$/g.test(dbSNP) &&
           <EntityPageVerticalTable
             title={<span><BookIcon style={{ marginRight: '1rem' }} /> External References</span>}
-            thToTd={
-              Object.keys(externalDbIds)
-              .filter(x => x !== '__dataID__')
-              .map(db => ({
-                th: db.replace(/_/g, ' '),
-                td: externalDbIds[db] && !!externalDbIds[db].length && (
-                  <ExternalLink
-                    href={externalReferenceLinks[db](externalDbIds[db][0])}
-                  >
-                    {externalDbIds[db]}
-                  </ExternalLink>
-                ),
-              }))
-          }
+            thToTd={[
+              { th: 'Name', td: 'dbSNP' },
```

Comment:
I think the ticket meant something like `{ th: 'dbSNP', td: dbSNP }`, so that we can add more in the future, unless there were more instructions outside of the ticket.